src/server/transaction.cc

```diff
@@ -621,26 +647,16 @@ void Transaction::ScheduleInternal() {
     };
     shard_set->RunBriefInParallel(std::move(cb), is_active);
 
-    if (success.load(memory_order_acquire) == num_shards) {
+    if (success.load(memory_order_acquire) == unique_shard_cnt_) {
       coordinator_state_ |= COORD_SCHED;
-      bool ooo_disabled = IsAtomicMulti() && multi_->mode != LOCK_AHEAD;
```

Comment:
what are the tx types that you allow now?

Reply:
We allow everything now. You allowed globals in https://github.com/dragonflydb/dragonfly/commit/bbe3d9303b769cfc244b0fbe72e86e4b1b944c5c#r137159159, so I guess it makes sense to allow multi globals as well. Actually, I don't remember why we disallowed them? There seems to be no real reason, because currently globals _can never return l_ock_granted = true from ScheduleInShard 🤔🤔